string filename
---
string STL
string OBJ

